src/nrc/tests/setup_configuration/test_notification_configuration.py

```diff
@@ -0,0 +1,64 @@
+import tempfile
+
+from django.core.management import call_command
+from django.test import TestCase
+
+from notifications_api_common.models import NotificationsConfig
+from zgw_consumers.models import Service
+
+
+class NotificationConfigurationTest(TestCase):
+    def test_notifications_api_service_identifier_is_optional(self):
+
+        config_yaml = """
+        notifications_config_enable: true
+        notifications_config:
+          notification_delivery_max_retries: 5
+          notification_delivery_retry_backoff: 2
+          notification_delivery_retry_backoff_max: 20
+        """
+
+        with tempfile.NamedTemporaryFile(
+            mode="w+", suffix=".yaml", delete=False
+        ) as tmp_file:
+            tmp_file.write(config_yaml)
+            tmp_file.flush()
+
+            call_command("setup_configuration", "--yaml-file", tmp_file.name)
```

Comment:
I'd prefer to have this tests similar to https://github.com/open-zaak/open-notificaties/blob/main/src/nrc/tests/setup_configuration/test_authorization_configuration.py#L25 for example, with a YAML file and executing the specific step instead of running the command via `call_command`. That way we don't have to generate tempfiles for this